+/- block tests failing with v11 beta

The +/- blocks build, but their tests fail.

```
1) List create block
       Xml
         Serialization
           append block
             0 items:
     AssertionError: expected 5 to equal undefined
      at assertListBlockStructure (webpack-internal:///./test/list_create.mocha.js:38:14)
      at Object.assertBlockStructure (webpack-internal:///./test/list_create.mocha.js:185:11)
      at Context.eval (webpack-internal:///../dev-tools/src/block_test_helpers.mocha.js:197:16)

  2) List create block
       Json
         Serialization
           append block
             0 items:
     AssertionError: expected 5 to equal undefined
      at assertListBlockStructure (webpack-internal:///./test/list_create.mocha.js:38:14)
      at Object.assertBlockStructure (webpack-internal:///./test/list_create.mocha.js:270:11)
      at Context.eval (webpack-internal:///../dev-tools/src/block_test_helpers.mocha.js:197:16)

...This continues with the same error...
```

--- a/plugins/block-plus-minus/package.json
+++ b/plugins/block-plus-minus/package.json
@@ -41,13 +41,13 @@
   "devDependencies": {
     "@blockly/dev-scripts": "^3.1.1",
     "@blockly/dev-tools": "^7.1.5",
-    "blockly": "^10.3.0",
+    "blockly": "^11.0.0-beta.2",
     "chai": "^4.2.0",
     "mocha": "^10.2.0",
     "sinon": "^9.0.1"
   },
   "peerDependencies": {
-    "blockly": "^10.3.0"
+    "blockly": "^11.0.0-beta.2"
   },
   "publishConfig": {
     "access": "public",

--- a/plugins/block-plus-minus/test/list_create.mocha.js
+++ b/plugins/block-plus-minus/test/list_create.mocha.js
@@ -35,7 +35,7 @@ suite('List create block', function () {
     if (inputCount === 0) {
       assert.equal(block.inputList.length, 1);
       const input = block.inputList[0];
-      assert.equal(input.type, Blockly.DUMMY_INPUT);
+      assert.equal(input.type, Blockly.inputs.inputTypes.DUMMY);
       assert.equal(input.name, 'EMPTY');
       assert.isNull(block.getField('MINUS'));
       // Easy way to test we're displaying empty instead of normal text.

--- a/plugins/block-plus-minus/test/procedures_test_helpers.mocha.js
+++ b/plugins/block-plus-minus/test/procedures_test_helpers.mocha.js
@@ -122,7 +122,7 @@ function assertDefBlockArgsStructure_(
   for (let i = 0; i < args.length; i++) {
     const expectedName = args[i];
     const defInput = defBlock.inputList[i + 1];
-    assert.equal(defInput.type, Blockly.DUMMY_INPUT);
+    assert.equal(defInput.type, Blockly.inputs.inputTypes.DUMMY);
     assert.equal(defInput.name, argIds[i]);
     assert.equal(
       defInput.fieldRow[2].getValue(),
@@ -136,7 +136,7 @@ function assertDefBlockArgsStructure_(
   const lastInput = defBlock.inputList[defBlock.inputList.length - 1];
   assert.notEqual(
     lastInput.type,
-    Blockly.DUMMY_INPUT,
+    Blockly.inputs.inputTypes.DUMMY,
     'last input is not a dummy',
   );
 }

--- a/plugins/block-plus-minus/test/text_join.mocha.js
+++ b/plugins/block-plus-minus/test/text_join.mocha.js
@@ -35,7 +35,7 @@ suite('Text join block', function () {
     if (!inputCount) {
       assert.equal(block.inputList.length, 1);
       const input = block.inputList[0];
-      assert.equal(input.type, Blockly.DUMMY_INPUT);
+      assert.equal(input.type, Blockly.inputs.inputTypes.DUMMY);
       assert.equal(input.name, 'EMPTY');
       assert.isNull(block.getField('MINUS'));
       // Easy way to test we're displaying quotes instead of normal text.

